.yarnrc.yml

```diff
@@ -0,0 +1,3 @@
+nodeLinker: node-modules
+
+yarnPath: .yarn/releases/yarn-4.1.0.cjs
```

Comment:
Forgive my ignorance re: yarn, but is this basically committing a specific version of the yarn executable into the repo for consistency?

Is `./yarn/install-state.gz` something that would get updated when the lock file updates?

Reply:
Correct, so everyone uses the same version, which will help avoid conflicts in the lock file, etc.

Re: `install-state.gz`, I don't consider myself a front-end guy either, so I had to look this up and learned that it shouldn't be checked in, actually, so I've removed it. See the changes in `.gitignore`